services/pull/protection_error.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
The file could be merged into above file and errors could be put on the top of the file since both are small.